tests/test_device.py

```diff
@@ -406,6 +406,20 @@ def test_args(self, mock_device):
         with pytest.raises(qml.DeviceError, match="specified number of shots needs to be at least 1"):
             Device(mock_device, shots=0)
 
+    def test_wire_map(self, mock_device):
```

Comment:
should update name here, since the function no longer exists :crying_cat_face:

Reply:
Nice one, thanks!